src/scenes/TradeHub/index.js

```diff
@@ -97,121 +128,129 @@ class TradeHub extends Component {
     }
   }
 
-  isHighestPrice(price) {
-    return price > this.state.highestPrice;
-  }
-
-  setHighestPrice(price) {
-    this.setState({
-      highestPrice: price
-    });
-  }
-
-  shouldSell(price) {
+  /**
+   * Price at which the current trading symbol is bought
+   */
+  setBoughtPrice = (price) => this.setState({ boughtPrice: price });
+
+  /**
+   * Difference between highestprice and sell price in percetages
+   */
+  setDiffPercentage = (percentage) =>  this.setState({ diffPercentage: percentage / 100 });
+
+  /**
+   * If the bot should sell
+   */
+  setSellEnabled = (value) => this.setState({ sellEnabled: value });
+
+  /**
+   * Check if price is higher then highest price
+   */
+  isHighestPrice = (price) => price > this.state.highestPrice;
+
+  /**
+   * Set highest price of current trading symbol since running
+   */
+  setHighestPrice = (price) => this.setState({ highestPrice: price });
+
+  /**
+   * Check if the bot should sell
+   */
+  shouldSell = (price) => {
     const { sellEnabled, hasSold, highestPrice, diffPercentage } = this.state;
     if (sellEnabled && !hasSold) {
       return price <= highestPrice - highestPrice * diffPercentage;
     }
   }
 
-  sell(price) {
+  /**
+   * Sell current trading symbol at given price
+   */
+  sell = (price) => {
     alert("SOLD at: " + price);
```

Comment:
Use single quotes

Reply:
The alert will be migrated to something like a toast message.